cmd/svcat/README.md

```diff
@@ -139,6 +139,29 @@ Additional parameters and secrets can be provided using the `--param` and `--sec
 --param p1=foo --param p2=bar --secret creds[db]
 ```
 
+You can also provide provision parameters in the form of a JSON string using the `--params` flag:
+
+```console
```

Comment:
A convention that I've started to follow for docs is:

*  use console block type when I am providing a command and its output, prefixing the command with `$`. This makes multi command blocks easy to read
* use an unspecified block type when I am showing just a command. That way commands are always in black, and console output in blue.